app/views/shared/_post.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class="row justify-content-center align-items-center mt-4">
+  <div class="col-lg-8 col-md-11 col-11 border p-2 ">
+    <h5>Post # <%= post_index %></h5>
+    <br>
+    <p><%= post.title %></p>
+    <p class="text-end">comments:  <%= post.comments_counter %> , likes:  <%= post.likes_counter %> </p>
+  </div>
+</div>
```

Comment:
Great job using partials, It's a very professional way to not repeat your code. 👍